db/migrate/20190805135829_backfill_metric_owners.rb

```diff
@@ -1,8 +1,8 @@
-require 'progress_counter'
+require Rails.root.join('db', 'data_migrations', '20200108171934_update_metric_owners')
 
 class BackfillMetricOwners < ActiveRecord::Migration
   def up
     Metric.reset_column_information
-    Rake::Task['services:update_metric_owners'].invoke
+    UpdateMetricOwners.new.up
```

Comment:
No need of this anymore right? If we remove the task?

Reply:
This migration won't be merged. It's just an example.